org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.raml

```diff
@@ -61,4 +61,53 @@ traits:
                   "value": false
                 }
               }
-                
+
+/SubscribeBinarySwitchResURI:
```

Comment:
You'll need to add postSubscribe*ResURI and deleteSubscribe*ResURI methods to the javascript.  the raml and javascript have to match, same pattern as getThermostatResURI